src/tedpy/ted5000.py

```diff
@@ -46,28 +46,32 @@ def gateway_description(self) -> str:
             "GatewayDescription"
         ]
 
-    def total_consumption(self) -> EnergyYield:
-        """Return consumption information for the whole system."""
+    def energy(self) -> EnergyYield:
+        """Return energy yield information for the whole system."""
         data = self.endpoint_data_results["LiveData"]
         power_now = int(data["Power"]["Total"]["PowerNow"])
         power_day = int(data["Power"]["Total"]["PowerTDY"])
         power_mtd = int(data["Power"]["Total"]["PowerMTD"])
-        return EnergyYield(YieldType.SYSTEM_NET, power_now, power_day, power_mtd)
+        return EnergyYield(power_now, power_day, power_mtd)
 
-    def mtu_value(self, mtu: TedMtu) -> MtuYield:
-        """Return consumption or production information for a MTU based on type."""
+    def _mtu_energy(self, mtu: TedMtu) -> EnergyYield:
```

Comment:
I wonder if we could be smart and total the MTU's for atleast generation(). Since the TED5000 interface has a bit more information in it I think we could do it fairly easily by looping over the MTUs. 

```
generation():
  for mtu in mtus:
    if(int(data["Power"]["MTU%d" % mtu.position]["PowerNow"]) < 0):
      total_gen += int(data["Power"]["MTU%d" % mtu.position]["PowerNow"])
```

You could do the same for Now, TDY, and MTD, if you thought it had value added.

```
        power_now = int(data["Power"]["MTU%d" % mtu.position]["PowerNow"])
        power_tdy = int(data["Power"]["MTU%d" % mtu.position]["PowerTDY"])
        power_mtd = int(data["Power"]["MTU%d" % mtu.position]["PowerMTD"])
```

Reply:
A bit better example of what I was trying to get across. Not exactly Python code but should get the idea across. I believe you want production to return a positive value. Otherwise remove the abs().

```python
    def production(self) -> EnergyYield:
        """Return generation information for the whole system."""
        gen = EnergyYield(0, 0, 0)
        for mtu in mtus:
          e = self._mtu_energy(mtu)
          if(min(e.now, e.daily, e.mtd) < 0):
            gen.now += abs(e.now)
            gen.daily += abs(e.daily)
            gen.mtd += abs(e.mtd)
        return gen
```

Reply:
I'm a bit confused how this would be helpful. Shouldn't the gen totals read from the API already be totalling the MTU? What would looping over the MTUs give us?

Reply:
> I'm a bit confused how this would be helpful. Shouldn't the gen totals read from the API already be totalling the MTU? What would looping over the MTUs give us?

This is for the TED5000 which does not have the DashData queries like the TED6000 does. In an effort to make a top level production() function for the TED5000 summing the values is an idea. Although now I see there is an MTU Type so that would be better than using the mtu_energy method.

`if mtu.type == MtuType.GENERATION:`